### ISSUE TITLE
Does not parse rule correctly

### ISSUE THREAD

**Issue by raguiar2** [teambition/RRuleSwift#16]
I don't think this library parses RRULE:FREQ=YEARLY;INTERVAL=1;COUNT=6;BYMONTH=2;BYDAY=3MO

(president's day)
correctly. I tried it, and it gives me back 
RRULE:FREQ=YEARLY;INTERVAL=1;WKST=MO;DTSTART=20190103T054718Z;COUNT=6;BYMONTH=2

which is incorrect. I think the issue is the BYDAY not parsing, but that's just a guess.

**Comment by dineshflock** [comment]
Yes, I have encounters this too. It parse `BYDAY=TU,TH` correctly but not `BYDAY=1FR`

**Comment by chriscborg** [comment]
Does anyone have a solution for this please?
